wandb/sdk/wandb_history.py

```diff
@@ -17,6 +19,7 @@ def __init__(self):
         self._data = dict()
         self._callback = None
         self._torch = None
+        self._start_time = time.time()
```

Comment:
Instead of history having it's own start_time we could pass the run object in here and use it but it felt a little funky.

Reply:
why cant we just put a settings object on history, then use self._settings._start_time?

Reply:
I think i get it... Normally settings would work for start_time. but with resume you are getting info too late, by that time settings was frozen so you are putting it into the run object.

I have some ideas to clean this up and can do them once this is checked in and I merge some of the `wandb sync` work